tests/lax_scipy_test.py

```diff
@@ -77,9 +77,11 @@ def op_record(name, nargs, dtypes, rng_factory, test_grad, test_name=None):
     op_record("entr", 1, float_dtypes, jtu.rand_default, False),
 ]
 
-if lib.version > (0, 1, 37):
```

Comment:
These can be `(0, 1, 37)` I think. (Same below.)

As it happens we are making a new Jaxlib release (0.1.38) right now, which will include both the beta and gamma functions you added. I expect we will immediately afterwards bump the minimum version to 0.1.38, at which point we can remove the `if` conditions.